### PR TITLE
covered content displayed in detailed view

### PR DESCRIPTION
**Pull request by kdrtibor**

### CODE REVIEW

File: src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/EntityDetailPresenter.java [hunk 1]
```diff
@@ -78,7 +79,11 @@ public void setEntity(Entity entityType, Long entityId) {
         RestUtil.runInBackground(
                 () -> {
                     try {
-                        return entityService.findEntity(entityType, entityId);
+                        EntityModel entityModel = entityService.findEntity(entityType, entityId);
```
Comment (atothhpe) [inline src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/EntityDetailPresenter.java]: Why must this be done by a separate rest call, why can't it be fetched with the original one?

Comment (atothhpe) [reply src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/EntityDetailPresenter.java]: Sets.newHashSet(Element) , but that's not relevant at the moment, it shouldn't be two rest calls

File: src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/GeneralEntityDetailsPanel.java [hunk 2]
```diff
@@ -307,8 +309,11 @@ private JXPanel updateUiWithTestsDetails(EntityModel entityModel, boolean isGher
                 getUiDataFromModel(entityModel.getValue(DetailsViewDefaultFields.FIELD_ESTIMATED_DURATTION)));
         testDetailsPanel.setLastModifiedDetails(
                 getUiDataFromModel(entityModel.getValue(DetailsViewDefaultFields.FIELD_LAST_MODIFIED)));
-        testDetailsPanel.setCoveredContentDetails(
-                getUiDataFromModel(entityModel.getValue(DetailsViewDefaultFields.FIELD_COVERED_CONTENT)));
+        String coveredContent = new String();
```
Comment (atothhpe) [inline src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/GeneralEntityDetailsPanel.java]: doesn't getUiDataFromModel() method work for this? there should be an util method to display this

File: src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/EntityDetailPresenter.java [hunk 3]
```diff
@@ -36,8 +43,9 @@
 import com.intellij.util.ui.ConfirmationDialog;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.Collection;
-import java.util.HashSet;
+import java.util.*;
```
Comment (alexandra-marinescu) [inline src/main/java/com/hpe/adm/octane/ideplugins/intellij/ui/detail/EntityDetailPresenter.java]: Why do you need the imports below if you do a star import anyway?
I prefer explicit imports for the sake of readability

**Pull request merged**